resources/lib/vrtapihelper.py

```diff
@@ -127,7 +127,7 @@ def tvshow_to_listitem(self, tvshow, program, cache_file):
         else:
             thumbnail = 'DefaultAddonVideo.png'
         if self._favorites.is_activated():
-            program_title = quote(tvshow.get('title').encode('utf-8'), safe=':/'.encode('utf-8'))
+            program_title = tvshow.get('title')
```

Comment:
We still have to quote forward slashes in titles, otherwise it messes up the routing.

Reply:
Okay, I see, I'll fix this.

Reply:
I already did last night, and the easiest was to create my own quote_slash()/unquote_slash() function.